### ISSUE TITLE
Test Dataset Evaluation

### ISSUE THREAD

**Issue by pankajcs033** [open-mmlab/OpenPCDet#862]
I have generate results for KITTI testset but while uploading, it is asking for .txt and .png files of 500MB (maximum) .zip format but .png files are generated are of more than 500MB alone and .png format in not meant to be compressed. So, I am unable to upload the results. please suggest me some solution. your help is highly appreciated. Thanks in advance.

**Comment by curiousboy20** [comment]
Hi, would you mind sharing how to generate results for KITTI test set? I tried running test.py --save_all but it seems the results are from the training dataset instead
